### ISSUE TITLE
TiCDC doesn't report any error when log disk is full

### ISSUE THREAD

**Issue by gozssky** [pingcap/tiflow#3362]
### What did you do?

1. Use `tiup playground` to deploy cluster with one TiCDC.
```bash
tiup --tag cdc-test playground --tiflash 0  --ticdc 1 v4.0.14
```
2. Stop cluster and enter the data directory.
```bash
cd .tiup/data/cdc-test
mv ticdc-0/ticdc.log .
mount -t tmpfs -o size=1M tmpfs ticdc-0
mv ticdc.log ticdc-0/ 
dd if=/dev/zero of=ticdc-0/zero  # should output "dd: writing to 'ticdc-0/zero': No space left on device"
```
3. Restart the cluster for several times.

### What did you expect to see?

Every time I restart the cluster, some new log like "Welcome xxx" should be outputted to the log file or report some error on `stdout` or  `stderr`.

### What did you see instead?

After a few attempts, log file doesn't update any more and there is no error report on `stdout` or `stderr`.

### Versions of the cluster

`v4.0.14` (I think it probably exists in all versions, but I only tested it with `v4.0.14`)

**Comment by gozssky** [comment]
It seems https://github.com/uber-go/zap/blob/aea2a5051d079dbe4606d990119c8b68f574ff59/options.go#L75 is not set.

**Comment by overvenus** [comment]
fixed by #3778
